tests/framework/microvm.py

```diff
@@ -509,8 +510,9 @@ def spawn(
 
         jailer_param_list = self.jailer.construct_param_list()
 
-        # Checking the timings requires DEBUG level log messages
-        self.time_api_requests = log_level == "Debug"
+        if self.time_api_requests and log_level != "Debug":
```

Comment:
do we need the `self.time_api_requests` part in the condition?

Reply:
no, we dont - I really need that vacation next week it seems :joy: